version/version.go

```diff
@@ -2,17 +2,31 @@ package version
 
 import (
 	"fmt"
+
+	"github.com/coreos/go-semver/semver"
 )
 
 // These constants follow the semantic versioning 2.0.0 spec (http://semver.org/)
-const (
+var (
 	major uint   = 1
 	minor uint   = 0
 	patch uint   = 0
 	meta  string = "beta"
 )
 
 var build string
+var semVer string
+
+func init() {
+	sv, err := semver.NewVersion(semVer)
```

Comment:
i didn't get your point if we gonna have multiple LTS version then i would suggest having distinct branch for every minor version at least like other like [ethereum](https://github.com/ethereum/go-ethereum/tree/release/1.0.1) does.

besides I'm just using semver package to safely parse the build time tag set on package so there be no need to manually change source for every release and via `git tag ` command and the ` -X github.com/zarbchain/zarb-go/version.semVer=$(VTAG)` ldflag it will be set automatically

Reply:
@d4rk5h0t

Yes, As you said we will have a separate and distinct branch per release (or tag).

In your code, Version depends on someone else (`git describe` for example). Right? Think crazy, maybe git doesn't work fine in this case. So we have trouble then.

My point is, when version defines manually, we have full control over that, even we can break the rules!

Reply:
you absolutely right 
but let say git fails or the tag is not following semver specification 
then as shown in code the `init()` function only runs only if tag setted on commit is parsable by semver package (semver standard) otherwise it will default to `v1.0.0-beta` as hardcoded in code witch I'll strongly recommend to set for every release branch

Reply:
This is a bit confusing. Git tag is not designed for versioning control and therefore better to not use them as a source of versioning.